compiler/src/iree/compiler/Codegen/Common/GPU/GPUVerifyDistribution.cpp

```diff
@@ -76,6 +76,13 @@ struct GPUVerifyDistributionPass final
               continue;
             }
 
+            // Backdoor: linalg.copy
+            if (isa<linalg::CopyOp>(op)) {
+              if (auto useDMAConfig =
+                      getLoweringConfig<IREE::GPU::UseGlobalLoadDMAAttr>(op))
```

Comment:
Removes one level of nesting
```suggestion
            if (isa<linalg::CopyOp>(op) && getLoweringConfig<IREE::GPU::UseGlobalLoadDMAAttr>(op)) {
```